src/helpers/sequelize.initializer.js

```diff
@@ -0,0 +1,3 @@
+import models from '../api/models';
+
+models.SequelizeMeta.truncate({ truncate: { cascade: false } });
```

Comment:
@nkpremices what does this do? Do you have a link to where you found this syntax?

Reply:
It is truncating the ```sequelizeMeta```  table.
Oh.. There was a time I had issues with ``Travis``` 
I added this file to resolve them by truncating that table. 
Now as I've fixed the issue by using a ```Travis database service``` I think we no longer need this file. I will just go and delete it.